ra/ra.go

```diff
@@ -1978,9 +2012,15 @@ func (ra *RegistrationAuthorityImpl) PerformValidation(
 		if prob != nil {
 			challenge.Status = core.StatusInvalid
 			challenge.Error = prob
-			go ra.countFailedValidation(vaCtx, authz.RegistrationID, authz.Identifier.Value)
+			go ra.countFailedValidation(vaCtx, authz.RegistrationID, authz.Identifier)
 		} else {
 			challenge.Status = core.StatusValid
+			bucketKey, err := ratelimits.NewRegIdDomainBucketKey(ratelimits.IssuancePausedPerDomainPerAccount, authz.RegistrationID, authz.Identifier.Value)
+			if err != nil {
+				ra.log.Warningf("Can't get domain bucket key for regID=[%d] authzID=[%s] err=[%s]",
+					authz.RegistrationID, authz.ID, err)
+			}
+			ra.limiter.Reset(ctx, bucketKey)
```

Comment:
I'd move this into a helper function and just say `go ra.resetAccountPausingLimit(vaCtx, authz.RegistrationID, authz.Identifier)` or something similar. That way it preserves symmetry with the `prob != nil` case above, and has the same context-cancellation (or lack thereof) properties too.